cmd/katib-controller/v1beta1/main.go

```diff
@@ -97,15 +97,15 @@ func main() {
 		MetricsBindAddress: metricsAddr,
 	})
 	if err != nil {
-		log.Error(err, "unable add APIs to scheme")
+		log.Error(err, "Failed to create the manager")
 		os.Exit(1)
 	}
 
 	log.Info("Registering Components.")
 
 	// Setup Scheme for all resources
 	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
-		log.Error(err, "Fail to create the manager")
+		log.Error(err, "unable add APIs to scheme")
```

Comment:
Let's be consistent with the first capital letter.
```suggestion
		log.Error(err, "Unable to add APIs to scheme")
```